src/aaudio/AudioStreamAAudio.h

```diff
@@ -108,6 +108,27 @@ class AudioStreamAAudio : public AudioStream {
         return oboe::Result::OK;
     }
 
+    oboe::Result notifyWorkloadIncrease(bool cpu, bool gpu, const char* debugName) override {
+        if (!isPerformanceHintEnabled()) {
+            return oboe::Result::ErrorInvalidState;
+        }
+        return mAdpfWrapper.notifyWorkloadIncrease(cpu, gpu, debugName);
```

Comment:
Does this have to be called from the callback thread?
Is the ADPF native API thread safe?